tools/depth-quality/depth-metrics.h

```diff
@@ -4,12 +4,15 @@
 // Plane Fit implementation follows http://www.ilikebigbits.com/blog/2015/3/2/plane-from-points algorithm
 
 #pragma once
+#include <algorithm>
 #include <vector>
 #include <mutex>
 #include <array>
-#include <imgui.h>
 #include <librealsense2/rs.hpp>
-#include "rendering.h"
+#include "float3.h"
+#include "plane.h"
+#define _USE_MATH_DEFINES
```

Comment:
Since this is a header file IMO we should protect multiple definitions by adding:
```
#ifndef _USE_MATH_DEFINES
#define _USE_MATH_DEFINES
#endif
#include <cmath>
```

Same as you can see in src/types.h

Reply:
Makes sense to me. Done.